localstack/ext/java/src/main/java/cloud/localstack/ServiceName.java

```diff
@@ -18,4 +21,30 @@
     public static final String CLOUDFORMATION = "cloudformation";
     public static final String CLOUDWATCH = "cloudwatch";
     public static final String SSM = "ssm";
+
+    private static Map<String, String> serviceMap = new HashMap<>();
+    public static String getServiceUrl(String service) {
+        return serviceMap.get(service);
+    }
+
+    static {
+        serviceMap.put(API_GATEWAY, "http://localhost:4567");
```

Comment:
I'd like to avoid hardcoding the service endpoints here. Is there any chance we can make `getEndpointOnWindows` implement a similar logic to `getEndpointOnLinux`? Can you elaborate a bit why the existing `getEndpoint(...)` method doesn't work under Windows? Thanks

If the same approach (calling a python process that reads the configuration) does not work under Windows, another option to get the default service ports/endpoints would be to read the `localstack_client` config file from the installation directory - typically `$TMPDIR/localstack_install_dir/.venv/lib/python*/site-packages/localstack_client/config.py` (where `python*` is a placeholder for the concrete python version, for example `python2.7` or `python3.6`).

The problem with using the default ports, though, is that this could lead to conflicts if users provide configuration settings that specify custom ports per service. Therefore, the dynamic port lookup (by calling the python process) would be the preferred option here.